css/my_website.css

```diff
@@ -0,0 +1,52 @@
+body{
+    margin:0px;
+    background-color:rgb(231, 231, 231);
+    color:rgb(65, 65, 65);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size:20px;
+    line-height:normal;
+}
+.main-header{
+    background-color:#003F87;
+    color:#ffffff;
+    height: 75px;
+    padding: 0em;
+    width:100%;
+    overflow: hidden;
+}
+.text{
+    color: black;
+    padding:1em;
+    margin-left:1em;
+    margin-right:1em;
+    outline-color: rgb(255, 0, 0);
```

Comment:
These different outline colors are kind of tough on the eyes. Think about what you want to emphasize and how rather than alternating outlines on every paragraph.

Reply:
Yeah so I decided to emphasize the life goals by using hearts emojies